components/api/db/queries.bal

```diff
@@ -70,8 +76,23 @@ public const string GET_ARTIFACT_FOR_USER_FROM_IMG_NAME_N_VERSION = "SELECT REGI
                                     AND REGISTRY_ARTIFACT.VERSION=? OR ORG_NAME IN (SELECT ORG_NAME FROM
                                     REGISTRY_ORG_USER_MAPPING WHERE USER_UUID = ? AND ORG_NAME = ?) AND IMAGE_NAME=? AND REGISTRY_ARTIFACT.VERSION=?";
 
-public const string GET_MEMBERS_ORG_USERS = "SELECT USER_UUID, USER_ROLE FROM REGISTRY_ORG_USER_MAPPING WHERE ORG_NAME = 
-                                    (SELECT ORG_NAME FROM REGISTRY_ORG_USER_MAPPING WHERE USER_UUID = ? AND ORG_NAME = ?) 
+public const string GET_MEMBERS_ORG_USERS = "SELECT USER_UUID, USER_ROLE FROM REGISTRY_ORG_USER_MAPPING WHERE ORG_NAME =
+                                    (SELECT ORG_NAME FROM REGISTRY_ORG_USER_MAPPING WHERE USER_UUID = ? AND ORG_NAME = ?)
                                     LIMIT ? OFFSET ?";
 
 public const string GET_MEMBERS_ORG_USERS_COUNT = "SELECT COUNT(USER_UUID) FROM REGISTRY_ORG_USER_MAPPING WHERE ORG_NAME = ?";
+
+public const string SEARCH_ORGS_TOTAL_COUNT = "SELECT COUNT(ORG_NAME) FROM REGISTRY_ORGANIZATION
+                                    WHERE REGEXP_LIKE (REGISTRY_ORGANIZATION.ORG_NAME, ?)";
+
+public const string SEARCH_ORGS_QUERY = "SELECT REGISTRY_ORGANIZATION.ORG_NAME, REGISTRY_ORGANIZATION.DESCRIPTION,
+                                    COUNT(REGISTRY_ORG_USER_MAPPING.USER_UUID) AS MEMBER_COUNT
+                                    FROM REGISTRY_ORGANIZATION LEFT JOIN
+                                    REGISTRY_ORG_USER_MAPPING ON REGISTRY_ORGANIZATION.ORG_NAME=REGISTRY_ORG_USER_MAPPING.ORG_NAME
+                                    WHERE REGEXP_LIKE (REGISTRY_ORGANIZATION.ORG_NAME, ?) GROUP BY REGISTRY_ORGANIZATION.ORG_NAME  LIMIT ? OFFSET ?";
```

Comment:
Do we need to use REGEXP_LIKE ? isn't LIKE enough for this ?. I am just asking for the sake of consistency. There are places where we have used LIKE. @nadundesilva  WDYT ?